mirdata/jams_utils.py

```diff
@@ -62,14 +71,39 @@ def jams_converter(
 
     jam = jams.JAMS()
 
+    # duration
+    duration = None
+    if audio_path is not None:
+        if os.path.exists(audio_path):
+            duration = librosa.get_duration(filename=audio_path)
+        else:
+            raise OSError(
+                'jams conversion failed because the audio file '
+                + 'for this track cannot be found, and it is required'
+                + 'to compute duration.'
+            )
+
```

Comment:
:star2:

Reply:
does line 78 potetntialltly raises errors? if so, better to document them